examples/lighting-app/nrfconnect/main/AppTask.cpp

```diff
@@ -307,18 +309,47 @@ void AppTask::FunctionHandler(AppEvent * aEvent)
     }
 }
 
+void AppTask::StartThread()
```

Comment:
You could probably extract this to a common function e.g. StartDemoThreadNetwork() in examples/platform/nrfconnect/util/ThreadUtil.h a reuse in both examples. Also you could use existing Kconfig settings in the code like CONFIG_OPENTHREAD_CHANNEL, CONFIG_OPENTHREAD_PANID etc. It may be a nice feature of our platform - being able to fully (or to high extent) configure it via Kconfig and not having to look for defines/constants (?)

Reply:
I agree with that, so only Mesh Prefix needs to be manually set here.

Reply:
I changed it, but anyway config values have to be copied, because of the Thread manual start mode. In the end I guess it looks a bit better than before.